### PR TITLE
Html

### PR DESCRIPTION
**Pull request by cloudwerewolf**

### CODE REVIEW

File: note.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var notelist = [{"title": "hi", "note": "test"}, {"title": "test", "note": "test"}, {"title": "test", "note": "hi"}];
+function displayNotes() {
+    for (var notes in notelist){
+        var div = document.getElementById("notepage");
+        var text = document.createElement("textarea");
+        text.cols = "45";
```
Comment (raymeskhoury) [inline note.js]: These should actually just be set as numbers rather than strings (though it doesn't impact the behavior).

File: note.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+var notelist = [{"title": "hi", "note": "test"}, {"title": "test", "note": "test"}, {"title": "test", "note": "hi"}];
+function displayNotes() {
+    for (var notes in notelist){
+        var div = document.getElementById("notepage");
```
Comment (raymeskhoury) [inline note.js]: This can be moved outside of the loop

File: note.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        var text = document.createElement("textarea");
+        text.cols = "45";
+        text.rows = "5";
+        text.id = "notelist";
+        text.value = notelist[notes].title + '\n' + notelist[notes].note;
```
Comment (raymeskhoury) [inline note.js]: Eventually we might want to have 2 separate elements for the note/title but this looks good for now.

File: index.html [hunk 4]
```diff
@@ -1,33 +1,24 @@
...
-        <meta name="description" content="NotePen">
         <link rel="stylesheet" href="main.css">
         <link href='https://fonts.googleapis.com/css?family=Scope One' rel='stylesheet'>
+        <script type="text/javascript"src="note.js"></script>
     </head>
     <body>
-        
-        <h1><center>NotePen</center></h1>
-
+        <h1 style="text-align:center">NotePen</h1>
```
Comment (raymeskhoury) [inline index.html]: The style here should move into css

File: index.html [hunk 5]
```diff
@@ -1,33 +1,24 @@
...
+            <input type="text" value="Title (optional)" id="title"><br><br>
+            <textarea cols="45" rows="5" id="note" draggable="false"></textarea><br>
+            <input type="Submit" value="Save" id="save" onclick="displayNotes()">
         </div>
-        
-        <div style="display: inline-block">
-            <center><input type=text name="Title" value="Title (optional)" id="title"></center><br>
-            <center><textarea name="Note" cols="45" rows="5" id="note" draggable="false"></textarea></center><br>
-            <input type="Submit" value="Save" id="save" onclick="saveNote(note.text, , )">
+        <br><hr><br>
```
Comment (raymeskhoury) [inline index.html]: I'm not sure what hr is exactly but I don't think we need it?

File: main.css [hunk 6]
```diff
@@ -58,11 +58,13 @@ input{
     background-color: cornsilk;
     border-color: rgba(184, 134, 11, 0.85);
 }
-#save {
+#save{
```
Comment (raymeskhoury) [inline main.css]: keep the space before { for consistency

File: index.html (hunk 5 above)
Comment (cloudwerewolf) [reply index.html]: it adds a line across the page, not needed but I like it

Comment (raymeskhoury) [reply index.html]: Ohh right, yeah let's keep it then :)

**Pull request closed without merging**